WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1006,9 +1006,11 @@ MiniCalDay-wrongMonth       = @MiniCalDay@  color: @SecondaryBrandingColor@;
 MiniCalDay-selected         = @MiniCalDay@  background-color: @hexToRGBA(SecondaryBrandingColor, 30)@;
 MiniCalDay-sched-selected   = @MiniCalDay-hilited@  border:1px solid black;
 
+MiniCalWeekNo               = @Text-anchor@; @Text-center@
+MiniCalWeekNoTitle          = @MiniCalDOWBg@ @Text-anchor@ @Text-center@ 
```

Comment:
can we use MiniCalWeekNo instead of repeating same styles here?

Reply:
Fixed: dde0967c152a80d298f2f1978d18fb71f424b6d3